cartpole_v0:


    state_size: 4
    n_actions: 2
    learning_rate: 0.01
    replay_memory_size: 100
    target_update_freq: 100
    initial_random_action: 1
    gamma: 0.99
    epsilon_decay: 0.99
    done_episode_reward: -200
    batch_size: 32
    max_iterations: 1000
    episodes: 500

    # Environment variables
    control_type: "velocity"
    min_pole_angle: -0.7
    max_pole_angle: 0.7
    max_base_velocity: 50
    max_base_pose_x: 1.0
    min_base_pose_x: -1.0


    running_step: 0.02
    pos_step: 1.0
    init_pos: 0.0
    wait_time: 0.1